x-pack/legacy/plugins/siem/public/pages/detection_engine/translations.ts

```diff
@@ -10,8 +10,16 @@ export const PAGE_TITLE = i18n.translate('xpack.siem.detectionEngine.pageTitle',
   defaultMessage: 'Detection engine',
 });
 
-export const PAGE_SUBTITLE = i18n.translate('xpack.siem.detectionEngine.pageSubtitle', {
-  defaultMessage: 'Last signal: X minutes ago',
+export const LAST_SIGNAL = i18n.translate('xpack.siem.detectionEngine.lastSignaltitle', {
```

Comment:
nit (and below keys as well):
```suggestion
export const LAST_SIGNAL = i18n.translate('xpack.siem.detectionEngine.lastSignalTitle', {
```